modules/apps/remote-web-app/remote-web-app-service/service.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.3.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_3_0.dtd">
+
+<service-builder auto-import-default-references="false" auto-namespace-tables="false" dependency-injector="ds" mvcc-enabled="true" package-path="com.liferay.remote.web.app">
+	<namespace>RemoteWebApp</namespace>
+	<entity local-service="true" name="RemoteWebAppEntry" remote-service="false" trash-enabled="false" uuid="true">
+
+		<!-- PK fields -->
+
+		<column container-model="true" name="entryId" primary="true" type="long" />
+
+		<!-- Group instance -->
+
+		<!--column name="groupId" type="long" /-->
```

Comment:
Right now registered remote web apps are global to the installation, but we should probably scope them by virtual instance, company, group, or whatever we decide.

Reply:
I'm not sure we want finer grained scope than just virtual instance or company. If we compare to deployed apps, for instance, they are all available globally, right?

I think it will depend on the SASS model we go with. If we plan to have multiple customers on one portal instance, we might want to use the same scoping mechanism used to hide customers data from each other (companyId maybe?).

@david-truong @jbalsas Thoughts?

Reply:
> I think it will depend on the SASS model we go with. If we plan to have multiple customers on one portal instance, we might want to use the same scoping mechanism used to hide customers' data from each other (companyId maybe?).

Yep, I listed this under "Open questions" ("In the future, if we move towards multi-tenancy, are we planning on using unique (sub)domains per tenant?") in [the research doc](https://docs.google.com/document/d/1L3AXKG4k229lN0w-IA0EWzOy-dflSUnGjuv-1E1w2cA/edit#heading=h.70o9d6bpqrgt). I was specifically talking about subdomains there, but in terms of persistence I would think that virtual instance would be the relevant scope to be thinking about.